src/AccessibleGooglePlacesAutocomplete.tsx

```diff
@@ -101,9 +99,30 @@ export class AccessibleGooglePlacesAutocomplete extends React.Component<
     this.getStatusNoResultsMessage = this.getStatusNoResultsMessage.bind(this);
   }
 
+  public onConfirm = (value: string) => {
+    this.predictions.forEach(element => {
+      if (element.description === value) {
+        const placesService = new google.maps.places.PlacesService(
+          document.getElementById('address_input')
+        );
+
+        placesService.getDetails(
+          { placeId: element.place_id },
+          (placeInfo: any, requestStatus: string) => {
+            if (requestStatus === 'OK') {
+              console.log(placeInfo);
```

Comment:
TODO: what's the best way to do this? Likely have a required `onConfirm` callback that gets fired here?